src/Helpers/SearchableFinder.php

```diff
@@ -97,7 +98,11 @@ private function getProjectClasses(): array
             $configFiles = Finder::create()->files()->name('*.php')->in($this->app->path());
 
             foreach ($configFiles->files() as $file) {
-                require_once $file;
+                try {
+                    require_once $file;
+                } catch (Error $e) {
+                    // The file could not be loaded due to an error. Continue.
```

Comment:
Do we have access to the console here? Maybe use `$command->info`? What do you think?

Reply:
Unless we would export `$command` as a global in the command itself (we shouldn't), `$command` will not be available here.

It's also not in the service container.

Reply:
I would just ignore these cases. The only real reason to log it to the user, would be to prevent unexpected  behaviour (classes not being scanned).

A reference in the docs seems good enough to me.

Reply:
You can access the command from here: https://github.com/algolia/scout-extended/blob/13bce7535928e7ec6e1b2c777c6c6cad675dea21/src/Helpers/SearchableFinder.php#L57.

Pass the command into the find method, and put it as private. The class is internal, so we can do those changes.

Reply:
Ah I completely missed that. Was in a bit of a rush, my apologies. Pushed a fix.